release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -114,6 +112,29 @@ self-signed certificates for authorization.
 
 //For more information, see Installing a cluster on OpenStack using your own infrastructure (link when published).
 
+[id="ocp-4-4-openstack-validates-rhcos-images-checksum"]
+==== OpenStack validates {op-system} images by checking sha256 checksum
+
+On {rh-openstack}, the installer now performs automatic checksum validation of
```

Comment:
Not critical, but suggest doing the long version of RHOSP. Maybe RHCOS too since there aren't the definitions nearby.